nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestProbesObject.java

```diff
@@ -39,28 +38,28 @@
 @Deploy("org.nuxeo.ecm.core.management")
 public class TestProbesObject extends ManagementBaseTest {
 
-    private static final String FAILURE_COUNT = "failure";
+    public static final String FAILURE_COUNT = "failure";
 
-    private static final String SUCCESS_COUNT = "success";
+    public static final String SUCCESS_COUNT = "success";
 
-    protected static final String RUN_COUNT = "run";
+    public static final String RUN_COUNT = "run";
 
-    protected static final String COUNTS = "counts";
+    public static final String COUNTS = "counts";
 
-    protected static final String ADMINISTRATIVE_STATUS = "administrativeStatus";
+    public static final String ADMINISTRATIVE_STATUS = "administrativeStatus";
 
-    protected static final String STATUS_INFOS = "infos";
+    public static final String STATUS_INFOS = "infos";
 
-    protected static final String STATUS_SUCCESS = SUCCESS_COUNT;
+    public static final String STATUS_SUCCESS = SUCCESS_COUNT;
 
-    protected static final String NAME = "name";
+    public static final String NAME = "name";
```

Comment:
Making them protected instead of private sure, but why public if not strictly needed?
Not that there is anything wrong with that :smiley:

Reply:
for records:
[reason](https://github.com/nuxeo/nuxeo-management-rest-api/pull/13#discussion_r337387271)
So to harmonize, I'm going public